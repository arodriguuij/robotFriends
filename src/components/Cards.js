import React from 'react';
import Card from './Card';

const Cards = ({ robots }) => {
    const cardsArray = robots.map((e, i) =>
        <Card key={i} id={e.id} name={e.name} email={e.email} />);
    return (
        <div>
            {cardsArray}
        </div>
    );
}

export default Cards;