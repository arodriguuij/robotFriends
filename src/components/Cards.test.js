import {shallow} from 'enzyme';
import React from 'react';
import Cards from './Cards';

it('expect to render Cards component', () => {
    const mockRobots=[
        {
            id: 1,
            name: 'John Snow',
            username: 'JohnJohn',
            email: 'john@gmail.com'
        }
    ]
    expect(shallow(<Cards robots={mockRobots}/>)).toMatchSnapshot()
}); 