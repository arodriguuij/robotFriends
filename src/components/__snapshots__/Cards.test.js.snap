// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Cards component 1`] = `ShallowWrapper {}`;
